hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/operations/AbstractCustomCreateOperation.java

```diff
@@ -90,6 +92,9 @@ protected AbstractCustomCreateOperation(
 
     @Override
     public OperationResult execute(@NonNull final MessageFrame frame, @NonNull final EVM evm) {
+        if (FrameUtils.isHookExecution(frame)) {
```

Comment:
Hmm since we're disabling `CREATE`/`CREATE2` here inside a hook execution, we can revert all the other changes in this file, right? (Since code at `0x16d` will never be executing the create opcodes.)